sys/net/application_layer/coap/gcoap.c

```diff
@@ -407,6 +408,11 @@ int gcoap_req_init(coap_pkt_t *pdu, uint8_t *buf, size_t len, unsigned code,
     hdrlen = coap_build_hdr(pdu->hdr, COAP_TYPE_NON, &token[0], GCOAP_TOKENLEN,
                                                      code,
                                                    ++_coap_state.last_message_id);
+#else
+    hdrlen = coap_build_hdr(pdu->hdr, COAP_TYPE_NON, NULL, GCOAP_TOKENLEN,
+                                                     code,
```

Comment:
minor nit: as RIOS sticks with Linux Kernel coding style if no explicitly stated otherwise, indention should be aligned with parentheses:

``` 
hdrlen = coap_build_hdr(pdu->hdr, COAP_TYPE_NON, NULL, GCOAP_TOKENLEN,
                        code, ++_coap_state.last_message_id);
```